nextcloudappstore/core/views.py

```diff
@@ -1,7 +1,36 @@
 from django.shortcuts import render
 from django.views.generic.base import TemplateView
+from django.views.generic.detail import DetailView
+from django.views.generic.list import ListView
 
-from nextcloudappstore.core.mixins import CategoryContextMixin, RecommendedAppsContextMixin
+from nextcloudappstore.core.mixins \
+     import CategoryContextMixin, RecommendedAppsContextMixin
+from nextcloudappstore.core.models import App, Category
 
-class HomeView(CategoryContextMixin, RecommendedAppsContextMixin, TemplateView):
+
+class HomeView(CategoryContextMixin,
+               RecommendedAppsContextMixin, TemplateView):
     template_name = "home.html"
+
+
+class AppDetailView(CategoryContextMixin, DetailView):
+    model = App
+    template_name = 'app/detail.html'
+    slug_field = 'id'
```

Comment:
No, it doesn't. I tried that and when getting DetailView complains that it needs either a pk or slug kwarg.